iot-commercial-application/src/main/resources/application.yml

```diff
@@ -90,7 +94,16 @@ spring:
     username: dcsa_db_owner
     password: 9c072fe8-c59c-11ea-b8d1-7b6577e9f3f5
 
+---
+# A profile for autoloading reference and test data
+spring:
+  config:
+    activate:
+      on-profile: loadtestdata
 
+dcsa:
+  dataloader:
+    autoload: [reference,sample]
```

Comment:
If it just load the same as the default there is no need for this section. Also badly named since it does not load test data

Reply:
Aaaah, i will remove it 👍

Reply:
It is removed now 👍

Reply:
The profile is still there, now it's just empty.

Reply:
Profile is removed now.